docs/topics/rn-new-features.adoc

```diff
@@ -7,21 +7,46 @@
 
-The current release supports the {ProductName} plugin for IntelliJ IDEA. The plugin provides assistance directly in IntelliJ IDEA for developers making changes for a migration or modernization effort.
+* Replacing `javax` artifact dependencies declared within `pom.xml` files
+* Replacing `javax` package import statements
+* Renaming `javax` XML schema namespaces
+* Renaming `javax` prefixed properties
+* Renaming bootstrapping files
 
-The plugin is available through the link:https://plugins.jetbrains.com[IntelliJ marketplace].
+Nearly all these new rules have quick fixes so the changes that they support can be automated when using an {ProductShortName} plugin or extension within an IDE (Red Hat CodeReady Studio, Eclipse, Eclipse Che, IntelliJ or Microsoft Visual Studio Code).
 
-.Rules for migrating from JBoss EAP 7.3 to 7.4
+The rules have two targets: `jakarta-ee:9` and `eap:8.` However, using the target `jakarta-ee` is sufficient for all of them.
 
-The current release contains rules for migrating from JBoss EAP 7.3 to 7.4. The target of these rules is `eap:7`.
+.OpenRewrite recipe support (Technology Preview)
 
-.Rules for migrating from JBoss EAP XP 2 to 3
+OpenRewrite is a tool that automates the application of source code changes through the execution of recipes.
+OpenRewrite recipes can now be executed using the {ProductShortName} CLI. The recipe shipped with Release 5.2.1 renames the import of `javax` packages to the corresponding `jakarta` equivalent.
 
-The current release contains rules for migrating from JBoss EAP XP 2 to 3 The target of these rules is `eapxp:3`.
+OpenRewrite is a Technology Preview feature only.
 
-[NOTE]
+[IMPORTANT]
 ====
-Rules for migrating from Thorntail to JBoss EAP XP 2 have been included since version 5.1.4. The target of these rules is `eapxp:2`.
+OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
 ====
+
+.MTA extension for Visual Studio Code compatibility with  Codespaces
+
+The MTA extension for Microsoft Visual Studio Code is now compatible with Microsoft Visual Studio Codespaces.
+
+.New rules for the Springboot to Quarkus migration path
+
+Several new rulesets were implemented that identify technologies now supported by Quarkus extensions. These rules identify the presence of the technology by a `.pom` dependency, an embedded `.jar` file, or a partial file path within a shaded Uber JAR.
```

Comment:
I think this is a bit repetitious and verbose. Also, it is a good idea to use the software's full name (re: JAR). For the file format extensions, IBM style guide says to use all caps (POM), no markup. OTOH, the RH Supp. guide uses backticks for extensions without actually stating that that is the way we mark up file extensions, so you can take your pick. Again, you do not need "new".

Suggestion: 

Rulesets identify Quarkus-supported technologies by detecting a POM dependency, embedded JAR file, or a partial file path [what the heck is that and how is it different from a full file path?] within an Apache Maven Shade uber JAR.